app/index.js

```diff
@@ -1,4 +1,5 @@
 import Vue from 'vue'
 import App from './app.vue'
 
+Vue.config.devtools = true
```

Comment:
Why we need devtools enabled?

Reply:
We can also use consola.addReporter to catch all fatals

Reply:
We dont, but had to test/debug the vue-app and with `yarn link` I still had to build the project to test it with Nuxt. And I thought, why not leave it on, its just a development tool anyway.

Reply:
It triggers an extra dev tool for end-users during loading. I think loading it may consume little CPU as it has to inspect events, ...

Reply:
If you are afraid that enabling it could cause problems I have no problem with removing this